content/cloud-docs/api-docs/providers.mdx

```diff
@@ -336,6 +726,559 @@ curl \
+| `:namespace`         | The namespace of the provider for which the platform is being deleted. For private providers this is the same as the `:organization_name` parameter                                                                  |
+| `:name`              | The name of the provider for which the platform is being destroyed                                                                                                                                                    |
+| `:version`           | The version for which the platform is being destroyed                                                                                                                                                                  |
+| `:os`                | The os of the provider platform that is being destroyed                                                                       |
+| `:arch`              | The architecture of the provider platform that is being destroyed                                                              |
+| `:registry_name`     | Must be `private`.                                                                                                                                                                                                   |
+
+[permissions-citation]: #intentionally-unused---keep-for-maintainers
+
+| Status  | Response                  | Reason                                                      |
+| ------- | ------------------------- | ----------------------------------------------------------- |
+| [204][] | Nothing                   | Success                                                     |
+| [403][] | [JSON API error object][] | Forbidden - public provider curation disabled               |
+| [404][] | [JSON API error object][] | Provider not found or user not authorized to perform action |
+
+
+### Sample Request 
+
+```shell
+curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request DELETE \
+  https://app.terraform.io/api/v2/organizations/hashicorp/registry-providers/private/hashicorp/aws/versions/3.1.1/platforms/linux/amd64
+```
+
+## Get a Provider
+
+`GET /organizations/:organization_name/registry-providers/:registry_name/:namespace/:name`
+
+### Parameters
+
+| Parameter            | Description                                                                                                 |
+| -------------------- | ----------------------------------------------------------------------------------------------------------- |
+| `:organization_name` | The name of the organization the provider belongs to.                                                       |
+| `:namespace`         | The namespace of the provider. For private providers this is the same as the `:organization_name` parameter |
```

Comment:
Should "parameter" actually be attribute here? If so, we should update this in all of the places where you have this line.

```suggestion
| `:namespace`         | The namespace of the provider. For private providers this is the same as the `organization_name` parameter. |
```

Reply:
The "Parameter" seciton here is referring to the path params (highlighted)
/organizations/`:organization_name`/registry-providers/`:registry_name`/`:namespace`/`:name`/versions/`:version`/platforms/`:os`/`:arch`


The attributes section under "Key path" are referring to the key/value pairs being passed in the body upon CREATE, UPDATE, PATCH requests. 

The GET and DELETE endpoints do not have that info. 

Hope that clarifies! I think the wording for "Key path" could be different but am leaving that to stay consistent with the existing API docs

Reply:
ahh okay gotcha - thank you!!